tests/lib/systems.sh

```diff
@@ -27,3 +27,34 @@ is_ubuntu_14_system(){
     fi
     return 1
 }
+
+get_architecture(){
+    local curr_arch
+    curr_arch="$(uname -m)"
+
+    case "$curr_arch" in
+    x86_64)
+        echo 'amd64'
+        ;;
+    i386)
+        echo 'i386'
+        ;;
+    armv7l)
+        echo 'armhf'
+        ;;
+    aarch64*)
+        echo 'arm64'
+        ;;
+    ppc64*)
+        echo 'ppc64el'
+        ;;
+    s390*)
```

Comment:
This is probably incorrect, it is either s390 or s390x, those are different architectures.

Reply:
This function has been removed.